src/areaCalculator/Shape.java

```diff
@@ -0,0 +1,9 @@
+package areaCalculator;
+
+import java.util.List;
+
+public interface Shape {
+
+    List saveFields();
+    void updateField(double area);
```

Comment:
No need for those two methods in Shape interface.

Reply:
removed